recipes/konoha/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
`LICENSE` is already included in the tarball from PyPI, so you don't need it--go ahead and remove from the PR.

Reply:
Just to confirm, should I remove the whole license section, or just this part?

Reply:
Update: Removed file and left _meta.yaml_ unchanged